src/target/riscv/riscv-013.c

```diff
@@ -2816,16 +2816,15 @@ void riscv013_fill_dmi_nop_u64(struct target *target, char *buf)
 	buf_set_u64((unsigned char *)buf, DTM_DMI_ADDRESS_OFFSET, info->abits, 0);
 }
 
-static int get_max_sbaccess(struct target *target)
+static uint32_t get_max_sbaccess(struct target *target)
```

Comment:
Why did you change this to uint32_t? Now the return -1 at the end is pretty dubious. I'd leave this as returning int.